spring-cloud-previews/google-cloud-networkconnectivity-spring-starter/src/main/java/com/google/cloud/networkconnectivity/v1/spring/PolicyBasedRoutingServiceSpringAutoConfiguration.java

```diff
@@ -1,222 +0,0 @@
-/*
```

Comment:
This corresponds to the client library breaking change from https://github.com/googleapis/google-cloud-java/pull/9036